backend/auth/middleware_test.go

```diff
@@ -0,0 +1,618 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
+
+package auth
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/omec-project/webconsole/configapi"
+	"github.com/omec-project/webconsole/dbadapter"
+)
+
+const (
+	SUCCESS_BODY = `{"Result":"Operation Executed"}`
```

Comment:
```suggestion
	SUCCESS_BODY = `{"Result":"Operation Executed"}`
```
Not a big issue but we can prefer camel case.